mavsdk/system.py

```diff
@@ -260,3 +274,5 @@ def cleanup():
             p.kill()
 
         atexit.register(cleanup)
+
+        return p
```

Comment:
It seems like this is never used. Or is it?

Reply:
it is used here: https://github.com/mavlink/MAVSDK-Python/pull/262/files#diff-533e5a3c2350d6297f370a3a339084b7065e00661f66016b08fe4d0585130262R78

It allow to retrieve the created thread to manage it later